cmake/setup_custom_targets.cmake

```diff
@@ -81,8 +87,8 @@ IF(NOT DEAL_II_COMPONENT_PACKAGE)
          ${CMAKE_COMMAND} -E echo ''
       && ${CMAKE_COMMAND} -E echo ''
       && ${CMAKE_COMMAND} -E echo '***************************************************************************'
-      && ${CMAKE_COMMAND} -E echo "**  Error: Could not generate binary package. The component is disabled."
-      && ${CMAKE_COMMAND} -E echo "**  Please reconfigure with -DDEAL_II_COMPONENT_PACKAGE=ON"
+      && ${CMAKE_COMMAND} -E echo '**  Error: Could not generate binary package. The component is disabled. **'
```

Comment:
@bangerth : from this line i would say the suggested usage is manual formatting for strings without variables and macro for strings with variables.